README.md

```diff
@@ -30,6 +30,18 @@ It will listen on <http://127.0.0.1:3030> by default.
 Note: If you have issues with `npm install` please update to npm 2.0+ using `npm install -g npm@2` 
 ([Issue #1594](https://github.com/mozilla/fxa-content-server/issues/1594))
 
+## Docker Based Development
+
+To run the content server via Docker, three steps are required:
+
+    $ docker build --rm -t mozilla/fxa_content_server
+    $ docker run --rm -v $PWD:/opt/fxa mozilla/fxa_content_server npm install
+    $ docker run --rm -v $PWD:/opt/fxa --net=host mozilla/fxa_content_server
+
+This method shares the codebase into the running container so that you can install npm and various modules required by package.json. It then runs FxA content server in a container, while allowing you to use your IDE of choice from your normal desktop environment to develop code.
+
+Be sure to copy server/config/local.json-dist to server/config/local.json per usual before the final docker invocation to run the service.
```

Comment:
Being new to docker, one thing I found strange is I had to access the content server via the ip address reported by:

```
> boot2docker ip
192.168.59.103
```

This means the config.json needs to be updated, as well as the [contentServer.url config item](https://github.com/mozilla/fxa-auth-server/blob/d5ae3b59cc1c7add09a3e70edfbe70b0f0321f74/config/dev.json#L3) in the fxa-auth-server.

Maybe make a note of this, or else show some instructions to newbies like me how I can access the docker container via localhost/127.0.0.1